amoro-ams/amoro-ams-server/src/test/resources/config.yaml

```diff
@@ -36,4 +36,37 @@ taskmanager:
       size: 1728m
 
 parallelism:
-  default: 1
\ No newline at end of file
+  default: 1
+
+containers:
+  - name: KubernetesContainer
+    container-impl: org.apache.amoro.server.manager.KubernetesOptimizerContainer
+    properties:
+      kube-config-path: "~/.kube/config"
+      image: apache/amoro:0.7-SNAPSHOT
```

Comment:
if image set, what **podTemplate.spec.image** level with config

Reply:
To make it easier to reflect the override process, I set the image version in podTemplate to `apache/amoro:0.6`, but the final version is still a`pache/amoro:0.7-SNAPSHOT`.